Pipeline/PackageBuildOutputs/PackageBuildOutputs.groovy

```diff
@@ -275,9 +306,15 @@ if (buildOutputsMap.size() == 0) {
 	println("** Copying build outputs to temporary package directory $tempLoadDir")
 
 	buildOutputsMap.each { deployableArtifact, info ->
-		String container = info[0]
-		Record record = info[1]
-		PropertiesRecord propertiesRecord = info[2]
+		String container = info.get("container")
+		String owningApplication = info.get("owningApplication")
+		Record record = info.get("record")
+		PropertiesRecord propertiesRecord = info.get("propertiesRecord")
+		DependencySetRecord dependencySetRecord = info.get("dependencySetRecord")
+		
+		if (props.generateSBOM && props.generateSBOM.toBoolean()) {
```

Comment:
Can we move the adding of the record to the SBOM after the copy was successful?
I think the "core" processing should happen first. Helps to read the code. I think to L380?